src/Desktop/UIAutomation/EventHandlers/EventListenerFactory.cs

```diff
@@ -540,10 +577,12 @@ public void UnregisterAutomationEventListener(int eventId,bool wait = false)
         /// </summary>
         public void UnregisterAllAutomationEventListners()
         {
+            #pragma warning disable CA2000 // Call IDisposable.Dispose()
```

Comment:
Inconsistent indents
```suggestion
#pragma warning disable CA2000 // Call IDisposable.Dispose()
```